include/stronk/can_pow.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include <cmath>
+#include <type_traits>
+
+namespace twig
+{
+
+template<typename StronkT>
+struct can_pow
+{
+    [[nodiscard]] constexpr auto pow(const auto& exponent) const noexcept -> StronkT
+    {
+        static_assert(std::is_floating_point_v<typename StronkT::underlying_type>);
+
+        return StronkT {std::pow(static_cast<const StronkT&>(*this).template unwrap<StronkT>(), exponent)};
+    }
+};
```

Comment:
might want to add a twig::pow freestanding function as well?

Reply:
Removed pow and sqrt instead - they are weird in terms of stronk types.